api_db.php

```diff
@@ -76,17 +76,18 @@
         $dbquery .= ' status, reply_type, reply_time, dnssec';
         $dbquery .= ' FROM query_storage q';
         $dbquery .= ' WHERE timestamp >= :from AND timestamp <= :until ';
-        if (isset($_GET['types'])) {
-            $types = $_GET['types'];
-            if (preg_match('/^[0-9]+(?:,[0-9]+)*$/', $types) === 1) {
+        if (isset($_GET['status'])) {
+            // if some query status should be excluded
+            $status = $_GET['status'];
+            if (preg_match('/^[0-9]+(?:,[0-9]+)*$/', $status) === 1) {
```

Comment:
This is just a suggestion, but I think it would make easier to understand the code if the variable was called `$excludedStatus` (lines 81, 82 and 88).

```suggestion
            $excludedStatus = $_GET['status'];
            if (preg_match('/^[0-9]+(?:,[0-9]+)*$/', $excludedStatus) === 1) {
```

Reply:
Sure, good idea.